src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -11,7 +11,7 @@
       "Size": 66533
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 441958
+      "Size": 500022
     },
```

Comment:
This is a Xamarin.Forms app (ported to .NET 6+), the size difference is not quite as noticeable on this one.